CRM/Member/Form/Membership.php

```diff
@@ -1356,6 +1356,10 @@ public function submit() {
     if ($this->_mode) {
       $params['total_amount'] = CRM_Utils_Array::value('total_amount', $formValues, 0);
 
+      //CRM-20264 : Store CC type and number (last 4 digit) during backoffice or online payment
```

Comment:
the fact this is inside $this->_mode gives me pause, also I feel we can use a parent function. The changes to Confirm & to Core_Payment look correct.